### PR TITLE
Create LIT6291HassabaRetu.xml

### PR DESCRIPTION
**Pull request by DenisNosnitsin1970**
New entry (I will include id into VAT et 69).

### CODE REVIEW

File: new/LIT6291HassabaRetu.xml [hunk 1]
```diff
@@ -0,0 +1,82 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng"
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6291HassabaRetu" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">ሐሳበ፡ ርቱዕ፡</title>
+                <title type="normalized" xml:lang="gez" corresp="#t1">Ḥassāba rәtuˁ</title>
```
Comment (thea-m) [inline new/LIT6291HassabaRetu.xml:9]: ʿ

File: new/LIT6291HassabaRetu.xml [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+                <title xml:lang="en" xml:id="t2">Computus of the Righteous</title>
```
Comment (thea-m) [inline new/LIT6291HassabaRetu.xml:10]: I would encode it as a translation of t1, not as a new title

File: new/LIT6291HassabaRetu.xml [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+          <div type="edition">
+            <note>The text follows <ref type="mss" corresp="BNFabb105"/>.</note>
+             <div type="textpart" subtype="incipit" xml:lang="gez">
+                 <ab>
+
+                   በስመ᎓ አብ᎓ ወወልድ᎓ ወመንፈስ፡ ቅዱስ፡ አሐዱ፡ አምላክ፡ ጸሐፍና፡ በዘንዜክር፡
```
Comment (thea-m) [inline new/LIT6291HassabaRetu.xml:59]: ጸሐፍነ?

File: new/LIT6291HassabaRetu.xml [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+                    ሐሳበ፡ ርቱዕ። በ፲ወ፩አበቅቴ፡ አመ፡ ፲ወ፬ለመስከረም፡ መጥቅዕ፡ በ፳ወ፪አበቅቴ፡
+                    አመ፡ ፰ለጥቀምት፡ መጥቅዕ፡ በ፫አበቅቴ፡ አመ፡ ፳ወ፯ለመስከረም፡ መጥቀዕ፨
+
+                   </ab>
+</div>
+<div type="textpart" subtype="explicit" xml:lang="gez">
+    <ab>
+      በሰሉሥ፡ ለእመ፡ ይሰርቅ፡ መስከረም፡ አመ፡ ፳ወ፫፡ ግበር፡ ምህልለ፡ እም፡ ፲ወ፱ኢተውሕድ፡ ወእመ፡ ፳ወ፬ኢተብዝሕ፨
```
Comment (thea-m) [inline new/LIT6291HassabaRetu.xml:67]: is ምህልለ correct?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

File: new/LIT6291HassabaRetu.xml (hunk 3 above)
Comment (DenisNosnitsin1970) [reply new/LIT6291HassabaRetu.xml:59]: yes

File: new/LIT6291HassabaRetu.xml (hunk 4 above)
Comment (DenisNosnitsin1970) [reply new/LIT6291HassabaRetu.xml:67]: it is

File: new/LIT6291HassabaRetu.xml (hunk 2 above)
Comment (DenisNosnitsin1970) [reply new/LIT6291HassabaRetu.xml:10]: yes.

File: new/LIT6291HassabaRetu.xml (hunk 1 above)
Comment (DenisNosnitsin1970) [reply new/LIT6291HassabaRetu.xml:9]: ok

**Review by thea-m — APPROVED** [review]

**Pull request merged**